src/webui/www/private/views/search.html

```diff
@@ -561,7 +561,7 @@
         };
 
         const stopSearch = function(searchId) {
-            const url = new URI('api/v2/search/stop');
+            const url = new URI('api/v2/search/delete');
```

Comment:
I pushed up a new commit with this small change. It should allow us to free up some memory when the search completes and seems like a straightforward change, but I could use a sanity check here.

Reply:
Removed this commit from this PR. I'm going to handle this differently in a future PR that will add support for restoring search tabs on page reload. That PR will instead delete search jobs with the tab is explicitly closed.